<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigForm</class>
 <widget class="QWidget" name="ConfigForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>429</width>
    <height>171</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>配置</string>
  </property>
  <widget class="QCheckBox" name="m_ck3D">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>26</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>三维</string>
   </property>
  </widget>
  <widget class="QLabel" name="m_lable">
   <property name="geometry">
    <rect>
     <x>266</x>
     <y>1</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>加载文件同时默认启动：</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="m_ck4Plane">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>56</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>切面</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="m_ckWL">
   <property name="geometry">
    <rect>
     <x>29</x>
     <y>58</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>切面默认窗宽位：</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="m_window">
   <property name="geometry">
    <rect>
     <x>54</x>
     <y>88</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;1528&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="m_level">
   <property name="geometry">
    <rect>
     <x>177</x>
     <y>88</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;862&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>W:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>159</x>
     <y>90</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>L:</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="m_ckColor">
   <property name="geometry">
    <rect>
     <x>54</x>
     <y>23</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>默认添加伪彩</string>
   </property>
  </widget>
  <widget class="QLabel" name="m_lableRedme">
   <property name="geometry">
    <rect>
     <x>19</x>
     <y>118</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>所有配置在下次加载数据生效</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_pbSave">
   <property name="geometry">
    <rect>
     <x>309</x>
     <y>88</y>
     <width>51</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>29</x>
     <y>28</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>三维</string>
   </property>
  </widget>
  <widget class="QLabel" name="m_lableRedme_2">
   <property name="geometry">
    <rect>
     <x>3</x>
     <y>150</y>
     <width>431</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string> 切面：ctrl /alt + 鼠标右键可以旋转/平移轴线；ctrl+s 恢复切面初始</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
